lib/presentation/mixins/media_picker_mixin.dart

```diff
@@ -96,6 +96,7 @@ mixin MediaPickerMixin on CommonMediaPickerMixin {
     });
 
     return await linagora_image_picker.ImagePicker.showImagesGridBottomSheet(
+      heightOfBottomSheet: SendFileDialogStyle.heightOfBottomSheet,
```

Comment:
with this, we can not expand the picker when select more. Do you have alternative solution?

Reply:
The picker can be scrolled

Reply:
Not good for me. Pls refer Telegram app it can be expanded height for picker.